app/forms/fields/address_field.py

```diff
@@ -0,0 +1,28 @@
+from functools import cached_property
+import logging
+
+from wtforms import FormField, Form, StringField
+
+logger = logging.getLogger(__name__)
+
+
+def get_form_class(validators):
+    class AddressForm(Form):
+        # Validation is only ever added to the 1 field that shows in all 4 variants
+        # This is to prevent an error message for each input box
+        line1 = StringField(validators=validators)
+        line2 = StringField()
+        town = StringField()
+        postcode = StringField()
+
+        @cached_property
+        def data(self):
+            return super().data
+
+    return AddressForm
+
+
+class AddressField(FormField):
+    def __init__(self, validators, **kwargs):
+        form_class = get_form_class(validators)
+        super().__init__(form_class, **kwargs)
```

Comment:
```suggestion
        super().__init__(form_class, **kwargs)
        
```

Reply:
Don't we lint on trailing new lines for Python files?

Reply:
Don't think so, if we do, it would be that there is one?.

Reply:
ok it must be a local setting for me